REST-API-project/new-book.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="new-book.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <title>New Book</title>
+  </head>
+  <body>
+    <div class="container">
+      <hr />
+      <h1>Create New Book</h1>
+      <hr />
+
+      <form action="http://localhost:3000/book" method="POST">
+        <div class="form-group">
+          <label for="ISBN">ISBN</label>
+          <input class="form-control" name="isbn" />
+        </div>
+
+        <div class="form-group">
+          <label for="Title">Title</label>
+          <input class="form-control" name="title" />
+        </div>
+
+        <div class="form-group">
+          <label for="Author">Author</label>
+          <input class="form-control" name="author" />
+        </div>
+
+        <div class="form-group">
+          <label for="PublishedDate">Published Date</label>
+          <input class="form-control" name="publishedDate" />
+        </div>
+
+        <div class="form-group">
+          <label for="Publisher">Publisher</label>
+          <input class="form-control" name="publisher" />
+        </div>
+
+        <div class="form-group">
+          <label for="numOfPages">Number of Pages</label>
+          <input class="form-control" name="numOfPages" />
+        </div>
+        <!--Other fields-->
+        <button type="submit" class="btn btn-primary">Submit</button>
+      </form>
+    </div>
+    <!-- bookstrap JS file -->
+    <script
+      src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
+      integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
+      integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
+      integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
Suggestion: Move the closing tag '>' to be at the end of line 69 so it is 'prettier'